### PR TITLE
Networking - nick

### PR DESCRIPTION
**Pull request by ChantalForget**
@SunsetBlade this look good?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+   On the Client, Run does nothing??????
+6. stop sets the value of thread to null after properly closing down all networkthreads
+
+Conclusion: To kill the thread, just set thread to null? What join() does
+is it halts execution until the thread is done looping (thread == null), so 
+some function must set that.
+
+More: The threading is used to allow the entity to run in the background rather than taking focus.
+
+wed 1.5 hrs = meeting
```
Comment (ChantalForget) [inline README.md:23]: could you remove your hours from the readme? :)

File: src/model/ServerModel.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+public class ServerModel {
+	
+	//I am the god of this game, despite this I am dumb.
+	//I do not know anything about any other part of the architecture.
+	//My methods are called by the ServerController who owns me. I act accordingly.
+	
+	//in theory, I will hold:
+		//- The master copy of the board
+		//- All character and player data
+		//- Basicly everything, I am god.
```
Comment (ChantalForget) [inline src/model/ServerModel.java:12]: :godmode:

Comment (SunsetBlade) [inline src/model/ServerModel.java:12]: yeah go for it, I can always fix it if something breaks

On Sun, Feb 15, 2015 at 1:54 PM, ChantalForget notifications@github.com
wrote:

> In src/model/ServerModel.java
> https://github.com/sstone93/MAGIC/pull/17#discussion_r24724838:
> 
> > @@ -0,0 +1,19 @@
> > +package model;
> > +
> > +public class ServerModel {
> > -   
> > -   //I am the god of this game, despite this I am dumb.
> > -   //I do not know anything about any other part of the architecture.
> > -   //My methods are called by the ServerController who owns me. I act accordingly.
> > -   
> > -   //in theory, I will hold:
> > -       //- The master copy of the board
> > -       //- All character and player data
> > -       //- Basicly everything, I am god.
> 
> [image: :godmode:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sstone93/MAGIC/pull/17/files#r24724838.

## 

Nicholas Laws

Comment (ChantalForget) [inline src/model/ServerModel.java:12]: you mean merge it? :D

Comment (SunsetBlade) [inline src/model/ServerModel.java:12]: yeah, is it this "Merge Pull Request" Button?

Comment (ChantalForget) [inline src/model/ServerModel.java:12]: yup :shipit:

**Pull request merged**